src/mantine-core/src/Text/Text.styles.ts

```diff
@@ -76,17 +79,17 @@ function getLineClamp(lineClamp: number): CSSObject {
   return null;
 }
 
-function getTruncate(truncate: boolean, truncateStart: boolean): CSSObject {
-  if (truncateStart) {
+function getTruncate({ theme, truncate }: GetTruncate): CSSObject {
+  if (truncate === 'start') {
     return {
       overflow: 'hidden',
       textOverflow: 'ellipsis',
       whiteSpace: 'nowrap',
-      direction: 'rtl',
-      textAlign: 'left',
+      direction: theme.dir === 'ltr' ? 'rtl' : 'ltr',
+      textAlign: theme.dir === 'ltr' ? 'right' : 'left',
     };
   }
-  if (truncate) {
+  if (truncate === 'end' || truncate === true) {
```

Comment:
Can be simplified to `if (truncate)` since both `'end'` and `true` are truthy values

Reply:
I can change it if you want, I dislike relying on truthy values since i think this is more clear.
If in the future we add another option for `truncate`, using truthy values could lead to problems